cucumber/api/features/authn_restricted_to.feature

```diff
@@ -0,0 +1,22 @@
+Feature: User and Host authentication can be network restricted
+
+  Background:
+    Given I am the super-user
+    And I successfully PUT "/policies/cucumber/policy/root" with body:
+    """
+    - !user
+      id: alice
+      restricted_to: 176.0.0.1
+
+    - !user
+      id: bob
+      restricted_to: [ "127.0.0.1", "172.0.0.0/8" ]
```

Comment:
How did you choose `172.0.0.0/8` here?

Reply:
I will be added a comment for this per Jonah's request. To summarize:
- `127.0.0.1` is to support running this test locally
- `172.0.0.0/8` is for running this test within docker-compose containers. The default docker bridge subnet is `172.17.0.0/16`. When docker-compose creates new networks, it changes the second octet, so `172.0.0.0/8` allows me to permit any connections in docker-compose environments for CI.

I had attempted to try and specify a more specific subnet in the docker-compose config for a more narrow restriction, but I was running into overlapping network issues and didn't have enough visibility into docker on the CI agents to dig into it further.